src/main/java/com/targomo/client/api/pojo/AbstractEdgeStatisticsCrossingRadiusOptions.java

```diff
@@ -0,0 +1,37 @@
+package com.targomo.client.api.pojo;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.targomo.client.api.enums.EdgeStatisticAggregationType;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.*;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@AllArgsConstructor
+@NoArgsConstructor
+@SuperBuilder
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public abstract class AbstractEdgeStatisticsCrossingRadiusOptions {
```

Comment:
we should refrain from putting hard coded default parameters in Options objects in the client